Pod/Classes/KolodaView/DraggableCardView/DraggableCardView.swift

```diff
@@ -214,9 +221,9 @@ public class DraggableCardView: UIView {
             transform = CATransform3DTranslate(transform, dragDistance.x, dragDistance.y, 0)
             layer.transform = transform
             
-            updateOverlayWithFinishPercent(dragDistance.x / CGRectGetWidth(frame))
+            updateOverlayWithFinishPercent(dragPercentage, direction:dragDirection)
             //100% - for proportion
-            delegate?.card(self, wasDraggedWithFinishPercentage: min(fabs(dragDistance.x * 100 / CGRectGetWidth(frame)), 100), inDirection: dragDirection)
+            delegate?.card(self, wasDraggedWithFinishPercentage: min(fabs(100 * dragPercentage), 100), inDirection: dragDirection)
```

Comment:
You calculate dragPercentage 2 times per one movement, better to create variable inside this scope and save value there.